Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
@mistydemeo Do you have to set the active developer directory to the CLT or is it good enough to simply have the CLT installed?

> Given that this only shifts the problem around, let's avoid applying the patch on El Cap.

👍. I'll tweak it locally before pushing to save the CI another two hour job.